storybook build currently fails

**Describe the bug**
A clear and concise description of what the bug is.
run `npm run storybook`

```
ERROR in ./src/stories/2-components/15-Charts.stories.js
Module not found: Error: Can't resolve '../../pages/dashboard/_utils' in '<...>/covid19-tracking-website/src/stories/2-components'
```

--- a/src/stories/2-components/15-Charts.stories.js
+++ b/src/stories/2-components/15-Charts.stories.js
@@ -3,7 +3,7 @@ import React from 'react'
 import AreaChart from '../../components/charts/area-chart'
 import BarChart from '../../components/charts/bar-chart'
 
-import { parseDate } from '../../pages/dashboard/_utils'
+import { parseDate } from '../../utilities/visualization'
 
 import usDaily from '../../../_data/v1/us/daily.json'
 

